lib/common/socket.c

```diff
@@ -462,6 +462,11 @@ void h2o_socket_setpeername(h2o_socket_t *sock, struct sockaddr *sa, socklen_t l
     memcpy(&sock->_peername->addr, sa, len);
 }
 
+int h2o_get_sock_fd(h2o_socket_t *sock)
```

Comment:
How about definiding the external function directly in evloop.c.h and in uv-binding.c.h, instead of having two layers (i.e. h2o_get_sock_fd and do_get_sock_fd)?